modules/openy_traction_rec_import/src/Commands/OpenyTractionRecImportCommands.php

```diff
@@ -199,11 +181,17 @@ public function cleanUp(array $options) {
    */
   public function fetch() {
     if (!$this->tractionRecFetcher->isEnabled()) {
-      $this->logger()->notice(dt('Fetcher is disabled!'));
+      $this->logger()->notice('Fetcher is disabled!');
       return FALSE;
     }
 
-    $this->tractionRecFetcher->fetch();
+    $this->logger()->notice("Fetching data from Traction Rec.");
+    $fetch = $this->tractionRecFetcher->fetch();
+
+    if (!is_dir($fetch)) {
+      $this->logger()->warning('Data fetch failed. Debug TractionRecClient::executeQuery for more info.');
```

Comment:
well, I bet we have all needed context here about actual issue for making this easier to digest in Live environment
Line 191 contains info that $fetch is not a dir, which means something. Logs  should be meaninfull with action items defined, or we just should avoid them.

Reply:
Lol yup. This was... a note to myself that I missed removing. I've added a logger statement in TractionRecClient that logs a meaningful error.